src/components/HdResponsive.vue

```diff
@@ -18,42 +16,46 @@ export default {
     return {
       matches: {},
       indeterminate: !matchMediaAvailable,
+      listeners: {},
     };
   },
   watch: {
     breakpoints() {
       this.setResponsiveFlags();
     },
   },
-  created() {
+  beforeMount() {
     this.setResponsiveFlags();
   },
-  mounted() {
-    this.addResizeEvents();
-  },
-  beforeDestroy() {
-    this.removeResizeEvents();
```

Comment:
I think we should consider: https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/removeListener

We are not removing the listeners anymore, right?

Reply:
The listeners are data of this component so they should be demolished by the garbage collector after the destruction.